packages/common/src/api/tan-query/useRemixes.ts

```diff
@@ -123,6 +123,7 @@ export const useRemixes = (
           { items: processedTracks }
         )
       )
+      dispatch(remixesPageActions.setCount({ count: data.count }))
```

Comment:
adding this back, i must've dropped it somewhere. but this is used for showing the # of remixes